amundsen_application/static/js/ducks/tableMetadata/api/tests/index.spec.ts

```diff
@@ -57,6 +57,9 @@ describe('helpers', () => {
   it('getTableDataFromResponseData', () => {
     Helpers.getTableDataFromResponseData(mockResponseData);
     expect(filterFromObjSpy).toHaveBeenCalledWith(tableResponseData, ['owners', 'tags']);
+    expect(Helpers.getTableDataFromResponseData(mockResponseData)).toMatchObject(
```

Comment:
Unit tests should test the purpose of a method, and the purpose of `getTableDataFromResponseData` is to filter `owners` and `tags` from the object. Since that purpose has nothing to do with `programatic_descriptions`, I would suggest removing this check.

Reply:
got it. Moved to its own test.

Reply:
It's not that it needs to be modified, more so that I don't see what needs to be tested based on the changes you're made. I'm not seeing any changes to the logic of `getTableDataFromResponseData`.

Reply:
ok, I guess I wanted to make sure that programmatic_descriptions were being returned in case somebody removed it from that method, but I could see how static typing takes care of that concern for us? I will remove it then.